Hide a link from the Startup Program page (#2037)
segmentio/segment-docs: src/guides/usage-and-billing/startup-program.md

The Segment Startup Program includes three components:

- Segment's **Startup Deal** - Participating startups receive $25,000* in annual credit toward our monthly [Team plan](https://segment.com/pricing/) for as long as they meet our eligibility requirements (up to 2 years).

> - Partner **Startup Deals** - We've partnered with other technology companies that offer valuable tools for startups to offer exclusive deals and promotions from marketing, data warehouse, and analytics tools.

Reviewer: ```suggestion
- Partner **Startup Deals** - Segment partners with other technology companies that offer valuable tools for startups to offer exclusive deals and promotions from marketing, data warehouse, and analytics tools.
```

Once you've been accepted to the Segment Startup Program, you can apply for the partner deals using [this form](http://bit.ly/segment-deal-redeem). (You can view a list of the available deals [here](https://bit.ly/segment-partner-deals).)

**How do I know if my accelerator/incubator/VC firm has a relationship with Segment?**

> Ask your program manager to see if they participate in the Segment Startup Program. If they do not, you can request that they [apply to become a partner](https://airtable.com/shr84MIvVo4k8xbaO){:target="_blank"}.

Reviewer: ```suggestion
Ask your program manager to see if they participate in the Segment Startup Program. If they don't, you can request that they [apply to become a partner](https://airtable.com/shr84MIvVo4k8xbaO){:target="_blank"}.
```
